neoglot.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+# !/usr/bin/env python3
```

Comment:
Remove the added space. This is not a regular comment, but a _shebang_, which tells Linux to execute it with python3 when you run the file as an executable.

Reply:
I see. Thank you! I have fixed that.